src/test/pages/TeacherRegistration.test.tsx

```diff
@@ -28,16 +33,32 @@ describe("Teacher Registration", () => {
     });
   };
 
-  it("On page load the teacher conditions are visible and accept button needs to be pressed in order to view the form", async () => {
+  it("it displays teacher confirmation screen, then teacher conditions, then registration form", async () => {
```

Comment:
Another really small one. Don't need to change this if you don't want, but the reason Jest gives you `it` as an alias of `test` is so that you can write test names that are readable in plain English. So this one could be:
```typescript
it("should display..."
```

So it reads as "It should display". The way it's done now it reads as "It it displays". I tend to write tests that start with "should", as it forces me to think of a good name, but you don't have to follow that pattern. But bear in mind that when you use "it", people are going to read that as part of the whole sentence. As I say, you don't have to use "should", for instance what you've done in a test below also works, as it reads perfectly:
```typescript
it("redirects to student registration page if user is not a teacher", async () => {
```

Reply:
Yeah that makes sense - edited several test names relating to registration and I'll keep an eye on this one as I edit or add to other tests.